### PR TITLE
New message types for getting documents

### PR DESCRIPTION
**Pull request by charleenfei**
Added GetDocumentRequest and GetDocumentResponse message types

### CODE REVIEW

File: p2p/p2p.proto [hunk 1]
```diff
@@ -32,6 +32,14 @@ message SignatureResponse {
   coredocument.Signature signature = 1;
 }
 
+message GetAnchoredDocumentRequest {
```
Comment (lucasvo) [inline p2p/p2p.proto]: I'd simplify the naming to:
```suggestion
message GetDocumentRequest {
```

File: p2p/p2p.proto [hunk 2]
```diff
@@ -32,6 +32,14 @@ message SignatureResponse {
...
+  bytes document_identifier = 1;
+}
+
+message GetAnchoredDocumentResponse {
```
Comment (lucasvo) [inline p2p/p2p.proto]:
```suggestion
message GetDocumentResponse {
```

File: p2p/p2p.proto (hunk 1 above)
Comment (charleenfei) [reply p2p/p2p.proto]: okay, will do!

**Comment by charleenfei** [comment]
closing this PR due to bugs with prototool/protoc

**Pull request closed without merging**